pkg/apis/config/v1alpha2/meshrootcertificate.go

```diff
@@ -122,6 +122,19 @@ type TresorCASpec struct {
 // can be (Active, Passive).
 type MeshRootCertificateIntent string
 
+const (
+	// ActiveIntent means the settings and certificate provider in this MRC are used for signing and
+	// validating certificates.
+	ActiveIntent MeshRootCertificateIntent = "active"
+
+	// PassiveIntent means the settings and certificate provider in this MRC are used for validating
+	// certificates.
+	PassiveIntent MeshRootCertificateIntent = "passive"
```

Comment:
I personally like passive but if there is strong agreement here, we should switch `active` to something like `signing`.

Reply:
I think ValidateOnly and Signing make understanding the code a bit more straightforward for newcomers

Reply:
I was initially leaning this way as well, but in that case does signing mean "signingAndValidating" since its doing both? The ambiguity and verbosity felt to much.  For this reason, after sitting on it for awhile and trying out several rotations I began to like `passive` and `active`

@jaellio you presented this at the servicemeshcon did you hear any feedback in either direction?